distributions/nr-otel-collector/manifest.yaml

```diff
@@ -2,7 +2,7 @@ dist:
   module: github.com/newrelic/opentelemetry-collector-releases/nr-otel-collector
   name: nr-otel-collector
   description: New Relic OpenTelemetry Collector
-  version: 0.6.0
+  version: 0.7.0
```

Comment:
Is this correct? We are not releasing anything new 🤔 I would wait until something is added and I would add to those release notes that the release adds support for Ubuntu 24.04.

Wdyt?

Reply:
I'm a bit afraid not to release now after some changes in the pipeline and then have to deal with some issues in the future , this is why i was planning to do a release